engine/shared/library/sharedFoundation/src/shared/NetworkId.h

```diff
@@ -68,8 +68,8 @@ class NetworkId
 
 	friend class DB::BindableNetworkId;
 
-  private:
-	explicit NetworkId(int value); // catch people who incorrectly try to construct one of these from an int
+//  private:
+//	explicit NetworkId(int value); // catch people who incorrectly try to construct one of these from an int
 	// also makes constructing one of these from any other incorrect numeric type ambiguous
```

Comment:
are we replacing this with some sort of verification that will throw an error if we pass an int to a NetworkId constructor otherwise we're just eliminating the type safety?